src/components/KTable/KTable.vue

```diff
@@ -544,9 +544,9 @@ const hasColumnVisibilityMenu = computed((): boolean => {
 // columns whose visibility can be toggled
 const visibilityColumns = computed((): TableHeader[] => tableHeaders.value.filter((header: TableHeader) => header.hidable))
 // visibility preferences from the host app (initialized by app)
-const visibilityPreferences = computed((): Record<string, boolean> => hasColumnVisibilityMenu.value ? props.tablePreferences.columnVisibility || {} : {})
+const visibilityPreferences = computed((): Partial<Record<string, boolean>> => hasColumnVisibilityMenu.value ? props.tablePreferences.columnVisibility || {} : {})
```

Comment:
I'm not sure I understand `Partial<Record<string, boolean>>` here

Isn't `Record<string, boolean>` equivalent?